src/controllers/dictionaryController.ts

```diff
@@ -73,15 +74,15 @@ export const updateSchema = async (req: Request, res: Response) => {
 
 export const diffDictionaries = async (req: Request, res: Response) => {
   const showReferences = req.query.references || false;
-  const name = req.query.name;
-  const leftVersion = req.query.left;
-  const rightVersion = req.query.right;
+  const name = req.query.name.toString();
+  const leftVersion = req.query.left.toString()
+  const rightVersion = req.query.right.toString();
 
   if (name && leftVersion && rightVersion) {
     const dict1Doc = await dictionaryService.findOne(name, leftVersion);
     const dict2Doc = await dictionaryService.findOne(name, rightVersion);
-    const dict1 = showReferences ? dict1Doc.toObject() : replaceReferences(dict1Doc.toObject());
-    const dict2 = showReferences ? dict2Doc.toObject() : replaceReferences(dict2Doc.toObject());
+    const dict1 = (showReferences ? dict1Doc.toObject() : replaceReferences(dict1Doc.toObject() as DictionaryDocument)) as DictionaryDocument;
+    const dict2 = (showReferences ? dict2Doc.toObject() : replaceReferences(dict2Doc.toObject() as DictionaryDocument)) as DictionaryDocument;
```

Comment:
a little messy but gets the job done. `toObject` changed it's return type from `any` to `LeanDocument<DictionaryDocument>` which is a mongoose lighter version of a full document which has internal state and other additions that cause bloat for basic usage.